RetailCoder.VBE/Inspections/InspectionsUI.de.resx

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
hrngh... fml this is really annoying. next commit with TranslationHelper will change this back again.